docs/core/development/review-process.rst

```diff
@@ -104,17 +104,6 @@ Authors: How to merge the change to trunk
  * Use the GitHub merge button to merge the request, using the GitHub default commit subject, and with the standard commit format required by Twisted. See below for details.
  * Alternatively, you can use the command line and merge the change into a checkout of Twisted trunk (as a merge commit, using `git merge --no-ff`) and commit it.
 
-The commit message, when using both the GitHub button or the CLI commit, must follow this format.
-This helps automatically close the associated ticket::
-
-    Merge pull request #123 from twisted/FIRST-LINE-AUTOMATICALLY-GENERATED-BY-GITHUB
-
-    Author: comma_separated_github_username
-    Reviewer: comma_separated_github_usernames
-    Fixes #issue number
-
-    Long description (as long as you wish)
-
```

Comment:
I'd probably leave in a note about how the ticket description or the commit message must say "Fixes #..." to trigger the automation.

It's good that the default ticket description includes this but sometimes people just delete the whole template and put in their own stuff.

Reply:
My goal was to reduce duplication... and the probability of getting documentation that is out of sync.

Also, the plan is to have an automated CI check for all this.

If your PR title has no ticket ID , the CI will fail
If the PR description has no `Fixes #1234` the CI will fail.

---------

Here I would keep only the general / high-level review  info.

All the low details can be written in the template and in the CI check.
And the template is just a file that is available.

---------

What we can do, is look into integrating the PR template inside the documentation.
So that we don't have to duplicate the content...and any change to the PR template will also automatically update the documentation.

Maybe this can be done as part of https://github.com/twisted/twisted/pull/11577